### ISSUE TITLE
highlighted terms not keyboard accessible

### ISSUE THREAD

**Issue by dianalam** [USDepartmentofLabor/ableist-language-detector-web#1]
Highlighted terms on the results page are not keyboard accessible (only mouse accessible). We need to update the highlight to be a button (or some other html element) so that it can be keyboard accessible: https://github.com/USDepartmentofLabor/ableist-language-detector-web/blob/master/highlight.py#L36 

> What It Means
An event handler is present that may not be accessible.
Why It Matters
The JavaScript events in use do not appear to be accessible to both mouse and keyboard users. To be fully accessible, critical JavaScript interaction should be device independent.
How to Fix It
Ensure that critical functionality and content is accessible by using a device independent event handler (which responds to both keyboard and mouse) or by using both a mouse dependent and a keyboard dependent event handler.
The Algorithm... in English
One of the following is present:
an onmouseover event but not an onfocus event
an onclick event on something other than a link, form control, or element with a tabindex value of 0
ondblclick
Standards and Guidelines
2.1.1 Keyboard (Level A)
